packages/flame/lib/src/events/component_mixins/tappable.dart

```diff
@@ -1,21 +1,24 @@
+import 'package:flutter/gestures.dart';
 import 'package:meta/meta.dart';
 
 import '../../../components.dart';
-import '../../game/mixins/has_tappables.dart';
 import '../../gestures/events.dart';
+import '../flame_game_mixins/has_tappables.dart';
 
+/// Mixin that can be added to any [Component] allowing it to receive tap
+/// events.
+///
+/// When using this mixin, also add [HasTappables] to your game, which handles
+/// propagation of tap events from the root game to individual components.
+///
+/// See [MultiTapGestureRecognizer] for the description of each individual
+/// event.
 mixin Tappable on Component {
```

Comment:
I think that the structure might be clearer with the mixins for components lives in `src/componentes/mixins/`, possibly with a sub directory for events in there. than having the `components_mixins` sub directory here. If we decide to change, I think that the structure should be changed for all the files at once, in a separate PR, so that we don't have some classes related to this in the `events` directory and some on other places in the intermediate time.

Reply:
Yeah, makes sense. I'll move the files back for now, and then we'll discuss the restructuring in a different PR.